src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -60,5 +64,13 @@
      * @return The human-readable name of this catalog type
      */
     String getName();
+    
+    @Override
+    default Text toText() {
```

Comment:
Didn't we have a discussion about `CatalogType`s being `TextRepresentable`? I distinctly remember having an issue with it, and I still do.